opencompass/utils/run.py

```diff
@@ -95,7 +95,14 @@ def get_config_from_arg(args) -> Config:
                      pad_token_id=args.pad_token_id,
                      run_cfg=dict(num_gpus=args.num_gpus))
         models.append(model)
-    return Config(dict(models=models, datasets=datasets),
+    
+    summarizer = None
+    if args.summarizer:
+        cfg = Config.fromfile(args.summarizer)
```

Comment:
```suggestion
        s = match_cfg_file('configs/summarizers/', [args.summarizer])[0]:
        get_logger().info(f'Loading {s[0]}: {s[1]}')
        cfg = Config.fromfile(s[1])
```
In this way, users can only pass the name of the summarizer config rather than the full path. (you may test it first)